AutoGrade/models.py

```diff
@@ -81,8 +81,9 @@ def student_email(self):
         return self.user.email
     student_email.short_description = 'Email'
 
-    def __str__(self):
+    """def __str__(self):
```

Comment:
Any specific reason for removing this? This is actually needed in operation.

Reply:
I forget to comment it out. 
It was causing some problems when i tried to access models from django admin to edit.